.github/workflows/ci.yml

```diff
@@ -82,7 +82,9 @@ jobs:
           build_dir=uncrustify/build
           cmake -S $source_dir -B $build_dir -G Ninja -DCMAKE_BUILD_TYPE=Release
           cmake --build $build_dir
-          mkdir -p $HOME/.cache
+          cmake --install $build_dir --prefix "$HOME/.local"
```

Comment:
Hmm, why is this needed?

Reply:
I can move this into another PR, it's just a small tweak for caching the uncrustify build more reliably.

Reply:
it probably was a separate commit at some point, but rebasing is annoying with multiple commits 🥲

Reply:
You did it again. Either learn git rebasing or create a new PR for this lol.